photoeditor/src/main/java/ja/burhanrashid52/photoeditor/PhotoEditorImpl.kt

```diff
@@ -192,6 +192,9 @@ internal class PhotoEditorImpl @SuppressLint("ClickableViewAccessibility") const
         return mGraphicManager.redoView()
     }
 
+    override val redoStackCount: Int
```

Comment:
If we change this to a boolean then the logic moves here :

```kotlin
 override val isRedoAvailable
      get() = mGraphicManager.redoViewsCount > 0
```
We need to add similar function for undo as well

Reply:
Done